src/App/contexts/taskContext.js

```diff
@@ -20,15 +22,48 @@ export const TaskProvider = (props) => {
     else setIsCompletedAll(true);
   };
 
+  // control loading screen
+  function triggerLoading() {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+    }, 900);
+  }
```

Comment:
I don't understand why you set loading is false after 900ms. What happens if the response from API takes longer than 900ms???

Reply:
I fixed this, now loading screen will end after fetch data process is completed